los_keeper/include/los_keeper/wrapper/wrapper.h

```diff
@@ -40,11 +40,13 @@ class Wrapper {
   DroneState drone_state_;
   store::State state_;
   std::vector<ObjectState> object_state_list_;
+  std::vector<ObjectState> target_state_list_;
```

Comment:
I thought object = target. This change will impact many lines

Reply:
I changed related stuffs. Don't worry.